src/hints/execution.rs

```diff
@@ -759,7 +759,7 @@ pub fn tx_account_deployment_data_len(
     let len = if tx.version.unwrap_or(Felt252::ZERO) < Felt252::THREE {
         0usize
     } else {
-        tx.account_deployment_data.unwrap().len()
+        tx.account_deployment_data.unwrap_or_default().len()
```

Comment:
Revert, we should fail if `account_deployment_data` is not specified here. This should actually return a `HintError::custom("tx.account_deployment_data must be specified".to_string().into_boxed_string())` instead of panicking, btw.

Reply:
In the meantime this change should be reverted though, it's incorrect to `unwrap_or_default()`.